src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,25 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 50px 40px;
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
```

Comment:
For a screen of 488px width, having only 2 columns of 200px each might not be the best use of space. Consider using 'auto-fill' or 'auto-fit' with a minmax function for better responsiveness.